srcpkgs/wine/template

```diff
@@ -57,6 +62,12 @@ if [ "$XBPS_LIBC" = "glibc" ]; then
 	hostmakedepends+=" prelink"
 fi
 
+if [ "${build_option_staging}" ]; then
+	do_patch() {
```

Comment:
Use post_patch instead, so any void patches would be used. Move `if` inside of function. Move function just before pre_build.